Presentation.Web/app/components/it-contract/it-contract-overview.controller.ts

```diff
@@ -618,6 +620,22 @@
                         sortable: false,
                         filterable: false
                     },
+                    {
+                        field: "LastChangedName", title: "Sidst redigeret: Bruger", width: 150,
+                        persistId: "lastchangedname",
+                        template: dataItem => `${dataItem.LastChangedByUser.Name} ${dataItem.LastChangedByUser.LastName}`,
+                        hidden: true,
+                        sortable: false,
+                        filterable: false
+                    },
+                    {
+                        field: "LastChanged", title: "Sidste redigeret: Dato", width: 150,
+                        persistId: "lastchanged",
+                        template: dataItem => dataItem?.LastChanged ? dataItem.LastChanged.toLocaleDateString() : "",
+                        hidden: true,
+                        sortable: true,
```

Comment:
this must be both filterable and sortable.. see how that is done in the it-system-catalog,controller.ts